tools/pipelines/build-build-tools.yml

```diff
@@ -114,7 +114,7 @@ extends:
     - test:copyresults
     taskLint: true
     taskLintName: ci:eslint
-    poolBuild: NewLarge-linux-1ES
+    poolBuild: Large-centralus
```

Comment:
We're actually not moving off them, the new ones are 1ES pools, I just dropped that bit from the name since we don't really need to differentiate between 1ES and non-1ES ones. We can run all pipelines on 1ES pools, whether they leverage 1ES templates or not.